### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by codewanderlust**
> Hello code reviewer, 🙋🏿‍♂️

In this pull request, I created methods to return the current users and forms  that aim to achieve the following;

- Create a Post on behalf of the current_user (use the method you created in your ApplicationController).
- Create a comment on behalf of the current_user (use the method you created in your ApplicationController).

I also added functionality that enables a user to add comments and likes to the posts

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def create
```
Comment (Ismailco) [inline app/controllers/comments_controller.rb:8]: - [ ] Greate work Implementing the `create` action for `comments_Controller` however you should include also the `new` action in your controller

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,11 @@
 Rails.application.routes.draw do
   root to: 'users#index'
+
+  get 'posts/new', to: 'posts#new'
+  post 'posts', to: 'posts#create'
+  post 'likes/:post_id', to: 'likes#create'
+  post 'comments/:post_id', to: 'comments#create'
```
Comment (Ismailco) [inline config/routes.rb]: - [ ] Kindly make sure to create a route for getting (`GET`) the comments
- [ ] Kindly make sure that the `get` and `post` methods for the `comment` and `like` are nested in the posts, you can do this easily by adding creating resources for `new` and `create` actions of `posts`, `comments`, and `likes`
e.g
```ruby
  resources :posts, only: [:create, :new] do
     resources :comments, only: [:create, :new]
     resources :likes, only: [:create]
  end
```

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @RoyJumah, 

Good job so far 👍🏼 
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @RoyJumah 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  - All changes requested have been implemented 🥇    

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**